README.md

```diff
@@ -428,6 +428,30 @@ machines, see the ["QEMU and Kubernetes"](#qemu-and-kubernetes) section below.
     $ kubectl label node <your node> storage=pmem
 ```
 
+- **Set up certificates**
+
+Certificates are required as explained in [Security](#security).
+If you are not using the test cluster described in
+[Starting and stopping a test cluster](#starting-and-stopping-a-test-cluster)
+where certificates are created automatically, you can set up certificates by
+running the `./test/setup-ca-kubernetes.sh` script manually in your cluster.
+This script requires "cfssl" tools which can be downloaded.
+These are steps for manual set-up of certificates:
+
+- Download cfssl tools:
+```sh
+   $ curl -L https://pkg.cfssl.org/R1.2/cfssl_linux-amd64 -o _work/bin/cfssl --create-dirs
+   $ curl -L https://pkg.cfssl.org/R1.2/cfssljson_linux-amd64 -o _work/bin/cfssljson --create-dirs
+```
+
+or,  can run above via `make _work/.setupcfssl-stamp`
```

Comment:
This "or, " looks odd. How about making it a separate sentence?

> Alternatively, one can run these commands via `make _work/.setupcfssl-stamp`.

Careful with the formatting. It'll look best if indendent under the "Download bullet point".

Reply:
Should we give "can run these via make target" note before the curl commands example? If user goes step-by-step, it's too late to show  (somewhat easier) alternative after user already performed those curl commands.

Reply:
> It is misleading only if user does not read full sentence. Start of sentence limits the scope to cases without test cluster.

`If you are not using the test cluster described in [Starting and stopping a test cluster](#starting-and-stopping-a-test-cluster) where certificates are created automatically, you must set up certificates by running the `./test/setup-ca-kubernetes.sh` script manually in your cluster.`

This sounds me like if its my own custom cluster, i must use this scritp. Isn't it?.

Reply:
Let's  break it down into smaller sentences.

> If you are not using the test cluster described in [Starting and stopping a test cluster](#starting-and-stopping-a-test-cluster) where certificates are created automatically, you must set up certificates manually. This can be done by running the ./test/setup-ca-kubernetes.sh script for your cluster.